.docker/app/Dockerfile

```diff
@@ -6,8 +6,8 @@ COPY . .
 
 USER root
 
-RUN apk add --update --no-cache bash=5.1.0-r0 tzdata=2021a-r0 libxslt-dev=1.1.34-r0 mariadb-dev=10.5.8-r0 && \
-    apk add --no-cache --virtual build-dependencies git=2.30.0-r0 build-base=0.5-r2 && \
+RUN apk add --no-cache bash=5.0.17-r0 tzdata=2020f-r0 libxslt-dev=1.1.34-r0 mariadb-dev=10.4.15-r0 nodejs=12.20.1-r0 yarn=1.22.4-r0 && \
```

Comment:
@sensei100 can we move nodejs and yarn to the --virtual build-dependencies section and then move line 17 where we delete build-depencies to be right before `rm -f /var/cache/apk/`?

Reply:
@dkinzer Okay, I pushed up that change.